src/tools/opensc-tool.c

```diff
@@ -553,6 +555,7 @@ static int list_algorithms(void)
 	int i;
 	const char *aname = "unknown";
 
+// clang-format off
```

Comment:
clang-format suggested the following change:


```
-// clang-format off
+	// clang-format off
```

<!-- identifier: clang-format -->
<!-- File: src/tools/opensc-tool.c -->